CONTRIBUTING.md

```diff
@@ -52,32 +52,34 @@ Here's a list of Lago's elements:
 - [lago](https://github.com/getlago/lago) - The entry point of the lago application
 - [lago/front](https://github.com/getlago/lago-front) - Lago's UI
 - [lago/api](https://github.com/getlago/lago-api) - Lago's API.
+- [lago/openAPI](https://github.com/getlago/lago-openapi) - OpenAPI definition for the Lago API
 
 #### The different clients
 
-- [lago/client/nodejs](https://github.com/getlago/lago-nodejs-client) - Lago's Nodejs Client
+- [lago/client/go](https://github.com/getlago/lago-go-client) - Lago's Go Client
+- [lago/client/javascript](https://github.com/getlago/lago-javascript-client) - Lago's Nodejs Client
 - [lago/client/python](https://github.com/getlago/lago-python-client) - Lago's Python Client (you are here 📍)
 - [lago/client/ruby](https://github.com/getlago/lago-ruby-client) - Lago's Ruby Client
 
 Also, because Lago is extensible, it's possible that a feature you've become accustomed to in Lago or an issue you're encountering isn't coming from a bundled package at all, but rather a community package you've installed. Each community package has its own repository too.
 
 ### Design Decisions
 
-If you have a question around how we do things, check to see if it is documented in the wiki of the related repository. If it is _not_ documented there, please open a new topic on [Lago Discussions](https://community.getlago.com/) and ask your question.
+If you have a question around how we do things, check to see if it is documented in the wiki of the related repository. If it is _not_ documented there, please open a new topic on [Lago Discussions](https://lago-community.slack.com) and ask your question.
 
 ## How Can I Contribute?
 
 ### Reporting Bugs
 
 This section guides you through submitting a bug report for Lago. Following these guidelines helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer: :computer:, and find related reports :mag_right:.
 
-Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out the required [template](https://github.com/getlago/lago-python-client/issues/new?assignees=&labels=%F0%9F%90%9E+bug&template=bug.md&title=%5BBUG%5D%3A+), the information it asks for helps us resolve issues faster.
+Before creating bug reports, please check [this list](#before-submitting-a-bug-report) as you might find out that you don't need to create one. When you are creating a bug report, please [include as many details as possible](#how-do-i-submit-a-good-bug-report). Fill out the required [template](https://github.com/getlago/lago-nodejs-client/issues/new?assignees=&labels=%F0%9F%90%9E+bug&template=bug_report.md&title=%5BBUG%5D%3A+), the information it asks for helps us resolve issues faster.
```

Comment:
Why does it switch to lago-nodejs-client ? Which is deprecated BTW

Reply:
Also concerns other occurrences bellow

Reply:
Oh! Good catch!